client/src/components/SignupForm.js

```diff
@@ -1,8 +1,8 @@
-import React, { useState } from 'react';
+import React, { useState, useContext } from 'react';
 import { Container, Form, FormGroup, FormText, Label, Input, Button } from 'reactstrap';
 import { gql, useMutation } from '@apollo/client';
 import { useHistory } from 'react-router-dom';
-import { localStorageSet } from '../lib/localStorage';
+import UserContext from '../providers/User'
```

Comment:
one question - i guess to @jwu910  is whether we should be importing in a relative way like this - which seems fine to me - or if it would be better to have some import roots defined etc?

Reply:
Relative imports are fine, more of a pattern and code style question.

Webpack allows you to set up [resolved paths](https://webpack.js.org/configuration/resolve/) for import aliases if you want to make it _look_ nicer, and not have to worry about something like `import Foo from '../../../../Bar';`